src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs

```diff
@@ -1049,5 +1380,214 @@ private static unsafe IntPtr CreateDefaultIReferenceTrackerTargetVftbl()
             vftbl[6] = (IntPtr)(delegate* unmanaged<IntPtr, uint>)&ComWrappers.IReferenceTrackerTarget_Unpeg;
             return (IntPtr)vftbl;
         }
+
+        [UnmanagedCallersOnly]
+        internal static unsafe int IReferenceTrackerHost_DisconnectUnusedReferenceSources(IntPtr pThis, uint flags)
+        {
+            try
+            {
+                HostServices.DisconnectUnusedReferenceSources(flags);
+                return HResults.S_OK;
+            }
+            catch (Exception e)
+            {
+                return Marshal.GetHRForException(e);
+            }
+
+        }
+
+        [UnmanagedCallersOnly]
+        internal static unsafe int IReferenceTrackerHost_ReleaseDisconnectedReferenceSources(IntPtr pThis)
+        {
+            try
+            {
+                HostServices.ReleaseDisconnectedReferenceSources();
+                return HResults.S_OK;
+            }
+            catch (Exception e)
+            {
+                return Marshal.GetHRForException(e);
+            }
+        }
+
+        [UnmanagedCallersOnly]
+        internal static unsafe int IReferenceTrackerHost_NotifyEndOfReferenceTrackingOnThread(IntPtr pThis)
+        {
+            try
+            {
+                HostServices.NotifyEndOfReferenceTrackingOnThread();
+                return HResults.S_OK;
+            }
+            catch (Exception e)
+            {
+                return Marshal.GetHRForException(e);
+            }
+
+        }
+
+        [UnmanagedCallersOnly]
+        internal static unsafe int IReferenceTrackerHost_GetTrackerTarget(IntPtr pThis, IntPtr punk, IntPtr* ppNewReference)
+        {
+            try
+            {
+                HostServices.GetTrackerTarget(punk, out *ppNewReference);
+                return HResults.S_OK;
+            }
+            catch (Exception e)
+            {
+                return Marshal.GetHRForException(e);
+            }
+        }
+
+        [UnmanagedCallersOnly]
+        internal static unsafe int IReferenceTrackerHost_AddMemoryPressure(IntPtr pThis, long bytesAllocated)
+        {
+            try
+            {
+                HostServices.AddMemoryPressure(bytesAllocated);
```

Comment:
E.g. we can call `GC.AddMemoryPressure(bytesAllocated);` directly here. No need to go through "HostServices".